packages/importer/src/index.ts

```diff
@@ -175,10 +175,25 @@ class OneSchemaImporter extends EventEmitter {
 
 export type OneSchemaImporterClass = InstanceType<typeof OneSchemaImporter>
 
+// this class is meant to shim the external facing class
+// for use in node based environments.
+const noop = () => null
+class MockOneSchemaImporter extends EventEmitter {
+  clientId: string = ""
+  iframe = {}
+  iframeConfig = {}
+  launch = noop
+  close = noop
+}
+
 export default function oneSchemaImporter(
   clientId: string,
   iframeConfig?: OneSchemaIframeConfig,
   baseUrl?: string,
 ): OneSchemaImporter {
-  return new OneSchemaImporter(clientId, iframeConfig, baseUrl)
+  if (typeof window !== "undefined") {
+    return new OneSchemaImporter(clientId, iframeConfig, baseUrl)
+  } else {
+    return new MockOneSchemaImporter() as unknown as OneSchemaImporter
```

Comment:
i have to do unknown because i'm not mocking the hidden properties `#...`, but i could mock those too.

we could also not have the mock object but in my mind it makes it safe to run the code on the server to see what happens..

open to other ideas but this feels like the ~ the right answer